src/support.js

```diff
@@ -0,0 +1,101 @@
+// support
+const data = require('./data');
+
+const animals = data.animals;
+
+const checkSex = (residents, option) => {
+  let message = '';
+  const array = [];
+  residents.forEach((resident) => {
+    if (resident.sex === option.sex) {
+      array.push(resident.name);
+    }
+    message = array;
+  });
+  if (option.sorted) {
+    message = array.sort();
+  }
+  return message;
+};
+
+const finalResidents = resident => resident.name;
+
+
+const otherResidentsChoice = (residents, option) => {
+  let message = '';
+  if (option.includeNames && (option.sex === 'male' || option.sex === 'female')) {
+    message = checkSex(residents, option);
+  } else {
+    message = residents.map(finalResidents);
```

Comment:
```suggestion
const otherResidentsChoice = (residents, option) => {
  let message = '';
  if (option.includeNames && (option.sex === 'male' || option.sex === 'female')) {
    message = checkSex(residents, option);
  } else {
    message = residents.map( resident => resident.name)
```
joga essa arrow dentro do map pra simplificar já que só tá sendo usada aqui